### ISSUE TITLE
Option to display shortened instrument names

### ISSUE THREAD

**Issue by sspagnol** [aodn/imos-toolbox#575]
Most (all?) instances references of instrument names are of a "make - model" variety which takes up a bit of screen realestate, especially in dialogs, example below

![Capture](https://user-images.githubusercontent.com/2917450/62999261-6fb9c080-bea0-11e9-90f3-41d2c6305b29.PNG)

When you have 15+ instruments it is hard to read.

I would like 
- an option (set in ini file) to use shortened names in display/dialogs
- shortened names defined in a simple an external file (essentially original and replacement text), example
```
"Seabird SBE56", "SBE56"
"Teledyne RDI Sentinel or Monitor Workhorse ADCP", "RDI"
"WET Labs WQM", 'WQM"
```

To reiterate this is only the display of instrument names in the toolbox, not the stored names in the netcdf file.

Maybe also in the ini file an ability to set font size to use in display/dialogs, for users with big hires monitors.

**Comment by ocehugo** [comment]
Requirements:

- [ ] A direct mapping between instrument/maker strings and a compressed representation
- [ ] There is a user switch to turn compressed/full string on/off.
- [ ] Revisit the FontSize and textual options for the GUI.

@sspagnol , this will be taken into account in the planning for the next release.

**Comment by ocehugo** [comment]
closed by aeaea02b353578444f7896998f343d22616aafcc
